code/modules/mob/living/living.dm

```diff
@@ -328,6 +347,24 @@
 
 		set_pull_offsets(M, state)
 
+/mob/living/proc/grab_output(mob/living/grabber, mob/living/grabbee, mod, grabtype)
```

Comment:
Whats the point of passing in grabber, if we are using src in the code anyway. If we are going to assume that grab_output is run on the mob being grabbed (like we are since src is used) then there is no point using grabber  too. Either replaces the srcs with grabber, or replace grabber with src.

Reply:
my B, resolved